dist/build_deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+IMAGE_BUILD="ekidd/rust-musl-builder:1.30.1"
```

Comment:
We might want to make these overridable as in `IMAGE_BUILD="${IMAGE_BUILD:-ekidd/rust-musl-builder:1.30.1}"`

Reply:
that's a nice to have, agree

Reply:
done